package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongo-unit",
-  "version": "1.4.5",
+  "version": "2.0.0",
```

Comment:
Breaking changes, right? Although, I have added backward compatibility.

Reply:
thank you for PR, I see mongo driver was updated
I updated the library and bumped dep versions

Reply:
V 1.5